core/domain/wipeout_domain.py

```diff
@@ -32,11 +34,12 @@ class PendingDeletionRequest:
 
     def __init__(
             self,
-            user_id,
-            email,
-            normalized_long_term_username,
-            deletion_complete,
-            pseudonymizable_entity_mappings):
+            user_id: str,
+            email: str,
+            normalized_long_term_username: Optional[str],
+            deletion_complete: bool,
+            pseudonymizable_entity_mappings: Dict[str, Dict[str, str]]
```

Comment:
Is this correct? The docstring types this as a dictionary of strings to strings. Does the docstring need to be updated?

Reply:
I think the docstring is wrong. Example of assignment in test:
https://github.com/oppia/oppia/blob/e8527d2ad18ebc6087372ae3d5b6cd76c8f9344f/core/domain/wipeout_service_test.py#L149-L151
It has the same structure as described in `PendingDeletionRequestModel` I think
https://github.com/oppia/oppia/blob/e8527d2ad18ebc6087372ae3d5b6cd76c8f9344f/core/storage/user/gae_models.py#L2754-L2772

Reply:
Done. I updated the docstring